static/src/javascripts/projects/commercial/modules/dfp/prepare-prebid.spec.ts

```diff
@@ -50,6 +51,10 @@ jest.mock('@guardian/consent-management-platform', () => ({
 	getConsentFor: jest.fn(),
 }));
 
+jest.mock('@guardian/libs', () => ({
+	log: jest.fn(),
+}));
```

Comment:
I think you can just use `spyOn` without having to mock this? Up to you.

Reply:
You're right but I think mixing spies and mocks can be a confusing distinction.

We generally don't use spies that much so will stick to codebase norms.